ftml/src/render/handle.rs

```diff
@@ -81,9 +81,9 @@ impl Handle {
 
         let (site, page, file): (&str, &str, &str) = match source {
             ImageSource::Url(url) => return Cow::clone(url),
-            ImageSource::File { file } => (&info.site, &info.page, &file),
-            ImageSource::OtherFile { page, file } => (&info.site, &page, &file),
-            ImageSource::RemoteFile { site, page, file } => (&site, &page, &file),
+            ImageSource::File1 { file } => (&info.site, &info.page, &file),
+            ImageSource::File2 { page, file } => (&info.site, &page, &file),
+            ImageSource::File3 { site, page, file } => (&site, &page, &file),
```

Comment:
I was writing out the JSON, and I thought it looked more clear to have the variants, since the fields are descriptive as to what this source is. "OtherFile" in isolation seemed a bit weird to me.

I can always change this later, in case there are strong feelings on this subject.